js/core/index.js

```diff
@@ -24,9 +24,23 @@ var keyboard = require('./keyboard');
 var ps2 = require('./ps2');
 var pci = require('./pci');
 var net = require('./net');
+var stdio = require('./stdio');
+var initio = require('./initio');
+
+function copy(source, dest) {
+  for (var obj in source) {
+    dest[obj] = source[obj];
+  }
+}
 
 function Runtime() {
-  this.tty = tty;
+  var self = this;
+  this.tty = {
+    StdOut: stdio.StdOut,
+    StdIn: stdio.StdIn
+  };
+  // Copy everything from `tty` to `this.tty`
+  copy(tty, this.tty);
```

Comment:
Sorry, i'd added more stuff and then I decided to delete them, guess I forgot about the copy.